bcgov/cas-registration
# 900 user profile name

https://github.com/orgs/bcgov/projects/122/views/2?pane=issue&itemId=53168930

to manually test:
- login as a bceid user
- change your first and last name in the profile page
- your name should update in the header
- go to the users table and you should the full updated name in the name column (previously it was getting cut off at last initial)
